beacon-chain/state/state-native/getters_attestation.go

```diff
@@ -19,11 +26,19 @@ func (b *BeaconState) PreviousEpochAttestations() ([]*ethpb.PendingAttestation,
 // previousEpochAttestationsVal corresponding to blocks on the beacon chain.
 // This assumes that a lock is already held on BeaconState.
 func (b *BeaconState) previousEpochAttestationsVal() []*ethpb.PendingAttestation {
+	if b.version != version.Phase0 {
```

Comment:
shouldnt we be returning an error here ? This will just return a 0 length slice.